src/ResourceManager/Advisor/Commands.Advisor/Microsoft.Azure.Commands.Advisor.format.ps1xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
```

Comment:
please create this file: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/generate-format.ps1xml-file.md

Reply:
ping.